### PR TITLE
Add receipt vars to lms_env_config

### PR DESCRIPTION
**Pull request by maxrothman**
As per https://openedx.atlassian.net/browse/DEVOPS-1084

### CODE REVIEW

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -124,6 +124,26 @@ EDXAPP_GIT_REPO_DIR: '/edx/var/edxapp/course_repos'
 
 EDXAPP_TENDER_DOMAIN: "None"
 
+EDXAPP_PDF_RECEIPT_BILLING_ADDRESS: |
```
Comment (feanil) [inline playbooks/roles/edxapp/defaults/main.yml]: Why pipe here and `>` below?

Comment (maxrothman) [inline playbooks/roles/edxapp/defaults/main.yml]: In the example given in the ticket, the address had newlines, while the disclaimer text did not.

Comment (feanil) [inline playbooks/roles/edxapp/defaults/main.yml]: Can you add a comment to that effect so someone doesn't try to fix it for consistency.

**Comment by feanil** [comment]
:+1:

**Comment by maxrothman** [comment]
According to @wedaly, the `EDXAPP_PDF_RECEIPT_COBRAND_LOGO_PATH` should be blank on edx.org, so I set it to an empty string. Should `EDXAPP_PDF_RECEIPT_LOGO_PATH` also be empty by default and be overridden, or is there a functional default?
@e0d @feanil @fredsmith opinions?

**Comment by feanil** [comment]
:+1:

**Pull request merged**